spring-kafka/src/main/java/org/springframework/kafka/annotation/ResolvableType.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2014-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.annotation;
+
+import org.springframework.util.MimeTypeUtils;
+
+import java.lang.annotation.*;
```

Comment:
We don't allow wildcard imports; please run `./gradlew check` to check style.

Reply:
Done